courses/machine_learning/deepdive/03_tensorflow/d_traineval.ipynb

```diff
@@ -9,12 +9,30 @@
    "source": [
```

Comment:
<p>I noticed that the reference to TensorBoard has been removed in the introductory paragraph, e.g. "We also use TensorBoard to monitor the training."  If we are to remove this reference, then we should also remove the heading "Monitor training" and aslo remove the cell that initializes the    OUTDIR = './taxi_trained'    variable, as that is what TensorBoard would use to build the graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>